### ISSUE TITLE
How to turn off Invalid Warning check

### ISSUE THREAD

**Issue by brodienguyen** [zgosalvez/github-actions-analyze-dart#40]
Hi there,

Please let me know if there is a way to skip this warning message. Btw, I wonder why it raises for those import lines, please?
<img width="1205" alt="image" src="https://user-images.githubusercontent.com/32996052/213868180-ea598278-3137-403c-827a-b562ab28a714.png">

Regards,
Brodie

**Comment by simplenotezy** [comment]
This is our workflow btw:

```yaml
name: Run checks

on:
  pull_request:
    types:
      - opened
      - reopened
      - synchronize
      - ready_for_review
env:
  FLUTTER_VERSION: "3.13.9"
  FLUTTER_CHANNEL: "stable"
  
jobs:
  build:
    name: Run checks
    runs-on: ubuntu-latest
    if: ${{ github.event.pull_request.draft == false }}
    steps:
      - name: Checkout code
        uses: actions/checkout@v2
        
      - name: Set up Flutter
        uses: subosito/flutter-action@v2.10.0
        with:
          flutter-version: ${{ env.FLUTTER_VERSION }}
          channel: ${{ env.FLUTTER_CHANNEL }}
          architecture: x64
          cache: true

      - name: Create .env
        run: echo "WHY_CANNOT_I_BE_EMPTY" > .env
      
      - run: |
          flutter pub get
          flutter pub run build_runner build --delete-conflicting-outputs

      - name: Analyze Dart
        uses: zgosalvez/github-actions-analyze-dart@v3
        with:
          line-length: 150
          fail-on-warnings: true
          fail-on-infos: false

**Comment by simplenotezy** [comment]
Perhaps it could be because line-length is not respected?

**Comment by zgosalvez** [comment]
@brodienguyen / @simplenotezy, care to verify if the PR will resolve your issue? If so, I will merge the changes.

**Comment by FriesI23** [comment]
> @brodienguyen / @simplenotezy, care to verify if the PR will resolve your issue? If so, I will merge the changes.

Hi, Thank you very much for providing this great Action first!

This PR([#96) is useful because generated files doesn't and shouldn't be formatted with `dart format`, e,g, `g.dart` by [json_serializable](https://pub.dev/packages/json_serializable#running-the-code-generator). 

These files also configured to be excluded in `analysis_options.yaml` normally in projects.  

I'm also feel confused by these warnings thrown from my `*.g.dart` files (generate by `flutter gen-l10n`, these files won't follow role defined by `flutter_lints` or `linter` option in `analysis_options.yaml`).

![image](https://github.com/zgosalvez/github-actions-analyze-dart/assets/20661034/399241e0-104c-405a-a793-0b1dc1dfcbca)
